addons/point_of_sale/models/uom.py

```diff
@@ -1,31 +1,34 @@
 from odoo import api, fields, models
 
 
-class UomCategory(models.Model):
-    _name = 'uom.category'
-    _inherit = ['uom.category', 'pos.load.mixin']
+class UomUom(models.Model):
+    _name = 'uom.uom'
+    _inherit = ['uom.uom', 'pos.load.mixin']
 
-    is_pos_groupable = fields.Boolean(string='Group Products in POS',
-        help="Check if you want to group products of this category in point of sale orders")
+    is_pos_groupable = fields.Boolean(string='Group Products in POS', compute='_compute_is_pos_groupable', store=True,
+        help="Check if you want to group products of this unit in point of sale orders")
 
-    @api.model
-    def _load_pos_data_domain(self, data):
-        return [('id', 'in', [uom['category_id'] for uom in data['uom.uom']])]
+    @api.depends('relative_uom_id')
+    def _compute_is_pos_groupable(self):
+        for uom in self:
+            uom.is_pos_groupable = uom.relative_uom_id.is_pos_groupable if uom.relative_uom_id else False
```

Comment:
Not sure this compute is relevant, even more since it only relies on its immediately-linked uom.

Reply:
I did it mainly for backward compatibility, so that customers have units that _were_ in the same category behave the same. Do you think it's better to do it in the upgrade script and make the field not computed, i.e. every unit will be independent of its reference?